client/my-sites/backup/main.jsx

```diff
@@ -202,11 +204,11 @@ function BackupStatus( {
 				) }
 
 				<BackupDatePicker onDateChange={ onDateChange } selectedDate={ selectedDate } />
-				<BackupStorageSpace />
+				<BackupStorageSpace onUsageLevel={ setUsageLevel } />
```

Comment:
@rustamshaikh I think maybe we can use a hook to manage the usageLevel state instead of passing it to multiple components. It might be useful for the next project as well.

What do you think?

Reply:
Thanks, @Initsogar for the suggestion. I will implement this using redux global state 👍